ocp_resources/datavolume.py

```diff
@@ -87,8 +87,39 @@ def __init__(
         yaml_file=None,
         delete_timeout=TIMEOUT_4MINUTES,
         api_name="pvc",
+        delete_after_completion=None,
         **kwargs,
     ):
+        """
+        DataVolume object
+
+        Args:
+            eviction (bool, default False): If True, set evictionStrategy to LiveMigrate explicitly.
+            name (str): DataVolume name.
+            namespace (str): DataVolume namespace.
+            source (str): source of DV - upload/http/pvc/registry.
+            size (str): DataVolume size - format size+size unit, for example: "5Gi".
+            storage_class (str, default: None): storage class name for DataVolume.
+            url (str, default: None): url for importing DV, when source is http/registry.
+            content_type (str, default: "kubevirt"): DataVolume content type.
+            access_modes (str, default: None): DataVolume access mode.
+            cert_configmap (str, default: None): name of config map for TLS certificates.
+            secret (Secret, default: None): secretRef.
+            client (DynamicClient): DynamicClient to use.
+            volume_mode (str, default: None): DataVolume volume mode.
+            hostpath_node (str, default: None): Node name to provision the DV on.
+            source_pvc (str, default: None): PVC name for when cloning the DV.
+            source_namespace (str, default: None): PVC namespace for when cloning the DV.
+            multus_annotation (str, default: None): network nad name.
+            bind_immediate_annotation (bool, default: None): when wffc sc is used and DV should be bind immediate.
+            preallocation (bool, default: None): preallocate disk space.
+            teardown (bool, default: True): Indicates if this resource would need to be deleted.
+            privileged_client (DynamicClient, default: None): Instance of Dynamic client
+            yaml_file (yaml, default: None): yaml file for the resource.
+            delete_timeout (int, default: 4 minutes): timeout associated with delete action.
+            api_name (str, default: "pvc"): api used for DV, pvc/storage
+            delete_after_completion (str, default: None): annotation for garbage collector, default behavior is "true"
```

Comment:
`default behavior is "true"` -where is this configured?

Reply:
by the cdi

Reply:
is it something an admin an change?

Reply:
`default behavior is "true"` I think you should remove this from the doc
we cannot update this based on CDI changes and it may not be true in later versions

Reply:
removed

Reply:
it is still there

Reply:
fixed